BodyPartElm[] body_part
uint32 person_id